src/Loader.php

```diff
@@ -325,9 +325,9 @@ protected function beginsWithAQuote($value)
     public function getEnvironmentVariable($name)
     {
         switch (true) {
-            case array_key_exists($name, $_ENV):
+            case isset($_ENV[$name]):
```

Comment:
The current behavior is "when $_ENV[$name] is null, return null" which is interpreted as "not found" in functions like `resolveNestedVariables()`. It seemed like the desired behavior should be to try each data source in the order provided to return a "found" value rather than exit earlier with a "not found" value. From what you said, it seems that `null` is a valid "found" value, which I find confusing, but can accept.